.redhat-ci.yml

```diff
@@ -59,3 +63,35 @@ tests:
 context: Clang
 
 artifacts:
+
+---
+
+inherit: true
+
+host:
+  distro: fedora/24/cloud
+
+context: rhci/vmcheck
+
+env:
+  VAGRANT_DEFAULT_PROVIDER: libvirt
+  VAGRANT_BOX_URL: https://ci.centos.org/artifacts/sig-atomic/centos-continuous/images/cloud/latest/images/centos-atomic-host-7-vagrant-libvirt.box
+  VAGRANT_BOX: centos/7/atomic/continuous
+
+# We'll need to add some sugar to redhat-ci to make this
+# nicer. E.g. allow provision multiple hosts, or a host and
+# a container.
+
+tests:
+  - dnf install -y vagrant libvirt qemu-kvm vagrant-libvirt ansible libselinux-python
+  - systemctl start libvirtd virtlogd
+  - vagrant box add --name $VAGRANT_BOX $VAGRANT_BOX_URL
+  - vagrant up
```

Comment:
This is going to be nested virt, which is going to be painful.  We should likely look at CentOS CI or another baremetal infrastructure for this.